.github/workflows/testing.yml

```diff
@@ -39,9 +39,9 @@ jobs:
       run: |
         source /opt/conda/bin/activate mssenv \
         && cd $GITHUB_WORKSPACE \
-        && pytest -v --durations=20 --cov=mslib mslib \
```

Comment:
Always running stuff in reverse is not helpful either. We would need to make two tests. Perhaps the reverse test can be a scheduled one (i.e. only weekly instead of once per commit).